### DISCUSSION TITLE
Anon Chat Enhancements

### DISCUSSION THREAD

**Discussion by Lylythii** [night/betterttv#5826] [Ideas]
Would be nice if Anon-chat could be toggled on/off with a button on the chat container, as an alternative to the `/join` command.

Additionally, whitelisting specific channels as "always logged in".
I find there's some streamers I'm always okay being seen by, where anon-chat is a mild irritant. & just prefer more "public" streams not seeing me.
Perhaps with a little new "`BetterTTV: [Anon Chat] You can be seen here!`" notice for whitelisted channels on initial stream opening/refresh.

**Comment by mynovelhost** [comment]
+1 for whitelist

**Comment by night** [comment]
Thanks for the suggestion. We've added a whitelist with commit 4f5d132e7a1b177df8ed8d99893b9c5e3a921b7a which will be out in the next release.
